campaignion_opt_in/campaignion_opt_in.module

```diff
@@ -47,9 +48,9 @@ function campaignion_opt_in_webform_conditional_form($node) {
       $element = [
         '#type' => 'select',
         '#options' => [
-          'yes' => t('Opt-in'),
-          '' => t('No change'),
-          'no' => t('Opt-out'),
+          Values::OPT_IN => t('Opt-in'),
+          Values::NO_CHANGE => t('No change'),
+          Values::OPT_OUT => t('Opt-out'),
```

Comment:
conditionals need not selected option too.

Reply:
Hm, thinking about this again and I don’t think it’s actually needed after all.